package.json

```diff
@@ -1,42 +1,45 @@
 {
-    "name": "ma_covid_vaccine_website",
-    "version": "0.1.0",
-    "private": true,
-    "dependencies": {
-        "@material-ui/core": "^4.11.3",
-        "@material-ui/icons": "^4.11.2",
-        "@testing-library/jest-dom": "^5.11.9",
-        "@testing-library/react": "^11.2.3",
-        "@testing-library/user-event": "^12.6.2",
-        "react": "^17.0.1",
-        "react-dom": "^17.0.1",
-        "react-loader": "^2.4.7",
-        "react-scripts": "4.0.1",
-        "react-sortable-table": "^1.4.0",
-        "web-vitals": "^0.2.4"
-    },
-    "scripts": {
-        "start": "react-scripts start",
-        "build": "react-scripts build",
-        "test": "react-scripts test",
-        "eject": "react-scripts eject"
-    },
-    "eslintConfig": {
-        "extends": [
-            "react-app",
-            "react-app/jest"
-        ]
-    },
-    "browserslist": {
-        "production": [
-            ">0.2%",
-            "not dead",
-            "not op_mini all"
-        ],
-        "development": [
-            "last 1 chrome version",
-            "last 1 firefox version",
-            "last 1 safari version"
-        ]
-    }
+  "name": "ma_covid_vaccine_website",
```

Comment:
Can you make sure you're using recently merged prettier config? I believe, unless if changed, that we are using 4 space indents. You can run prettier on the whole workspace if you wish.